pkg/operator/operatorclient.go

```diff
@@ -0,0 +1,36 @@
+package operator
+
+import (
+	"fmt"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
nit: split system and k8s deps with a newline

Reply:
I see @s-urbaniak has commented the other file because there was a blank line :)

@stlaz @s-urbaniak @sttts Do we have a formatter for our IDEs somewhere or at least written guidelines about the code style? Sorting imports manually is a wasteful job. 

If there is no formatter and no written code style guidelines, I suggest to leave this as is and introduce the `goimports` into our Makefiles (of course I volunteer to do that).

Reply:
It turned out that @stlaz uses `goimports`. I'll give it a go.